### PR TITLE
feat(route): add route /hottoys (#16238)

### LINKED ISSUE #16238: hottoys.com.hk 感謝!!
### Category

ACG

### Website URL

https://www.hottoys.com.hk/

### Website description

Hot Toys於2000年創立，公司總部在香港，主要製作1：6的美軍特種部隊可動人偶。之後，產品製作方向,漸漸轉向將版權作品之角色實體化的高品質收藏人形。

### What content should be included?

主頁中post的rss
非常感謝各位大神!!

### Additional description

_No response_

### This is not a duplicated rss request

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) and [pull requests](https://github.com/DIYgod/RSSHub/pulls) to ensure this rss proposal has not already been requested

### PR DESCRIPTION
**Pull request by jw0903**
## Involved Issue / 该 PR 相关 Issue

Close #16238

## Example for the Proposed Route(s) / 路由地址示例

```routes
/hottoys
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [x] `Puppeteer`

### CODE REVIEW

File: lib/routes/hottoys/index.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    path: '/',
+    categories: ['shopping'],
+    example: '/hottoys',
+    radar: [
+        {
+            source: ['hottoys.com.hk/'],
+        },
+    ],
+    name: 'hottoys',
+    maintainers: ['@lin'],
```
Comment (TonyRL) [inline lib/routes/hottoys/index.ts:15]:
```suggestion
    maintainers: ['jw0903'],
```

File: lib/routes/hottoys/namespace.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: 'hottoys',
```
Comment (TonyRL) [inline lib/routes/hottoys/namespace.ts:4]:
```suggestion
    name: 'Hot Toys',
```

File: lib/routes/hottoys/index.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+export const route: Route = {
+    path: '/',
+    categories: ['shopping'],
+    example: '/hottoys',
+    radar: [
+        {
+            source: ['hottoys.com.hk/'],
+        },
+    ],
+    name: 'hottoys',
```
Comment (TonyRL) [inline lib/routes/hottoys/index.ts:14]: Route name shouldn't be the same as namespace's name

File: lib/routes/hottoys/index.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import puppeteer from '@/utils/puppeteer';
+
+export const route: Route = {
```
Comment (TonyRL) [inline lib/routes/hottoys/index.ts:5]: Please set [`features.requirePuppeteer`](https://docs.rsshub.app/joinus/new-rss/start-code#creating-route) to true.

Comment (jw0903) [reply lib/routes/hottoys/index.ts:5]: fixed

**Pull request merged**